lib/beaker-hostgenerator/data.rb

```diff
@@ -51,5 +61,7 @@ module Data
         'consoleport' => 443,
       }
     }
+
+    module_function :pe_version, :pe_family, :pe_upgrade_version, :pe_upgrade_family
```

Comment:
the only place i see this module used is: test/util/generator_helpers.rb
why not just mix this module in?  particularly if the above file ends up with multiple modules/classes.